docs/source/llms/tracing/index.rst

```diff
@@ -0,0 +1,615 @@
+    )
+
+
+If we navigate to the MLflow UI, we can see not only the model that has been auto-logged, but the traces as well, as shown in the below video:
+
+.. figure:: ../../_static/images/llms/tracing/langchain-tracing.gif
+    :alt: LangChain Tracing via autolog
+    :width: 100%
+    :align: center
+
+.. note::
+    The example above is purposely simple (a simple chat completions demonstration) for purposes of brevity. In real-world scenarios involving complex 
+    RAG chains, the trace that is recorded by MLflow will be significantly more complex and verbose. 
+
+
+Tracing Fluent APIs
+-------------------
+
+MLflow's :py:func:`fluent APIs <mlflow.start_span>` provide a straightforward way to add tracing to your functions and code blocks. 
+By using decorators, function wrappers, and context managers, you can easily capture detailed trace data with minimal code changes. 
+
+As a comparison between the fluent and the client APIs for tracing, the figure below illustrates the differences in complexity between the two APIs, 
+with the fluent API being more concise and the recommended approach if your tracing use case can support using the higher-level APIs.
+
+.. figure:: ../../_static/images/llms/tracing/fluent-vs-client-tracing.png
+    :alt: Fluent vs Client APIs
+    :width: 60%
+    :align: center
+
+This section will cover how to initiate traces using these fluent APIs.
+
+Initiating a Trace
+^^^^^^^^^^^^^^^^^^
+
+In this section, we will explore different methods to initiate a trace using MLflow's fluent APIs. These methods allow you to add tracing 
+functionality to your code with minimal modifications, enabling you to capture detailed information about the execution of your functions and workflows.
+
+Trace Decorator
+###############
+
+The trace decorator allows you to automatically capture the inputs and outputs of a function by simply adding the :py:func:`@mlflow.trace <mlflow.trace>` decorator 
+to its definition. This approach is ideal for quickly adding tracing to individual functions without significant changes to your existing code.
+
+.. code-block:: python
+
+    import mlflow
+
+    # Create a new experiment to log the trace to
+    mlflow.set_experiment("Tracing Demo")
+
+
+    # Mark any function with the trace decorator to automatically capture input(s) and output(s)
+    @mlflow.trace
+    def some_function(x, y, z=2):
+        return x + (y - z)
+
+
+    # Invoking the function will generate a trace that is logged to the active experiment
+    some_function(2, 4)
+
+You can add additional metadata to the tracing decorator as follows:
+
+.. code-block:: python
+
+    @mlflow.trace(name="My Span", span_type="func", attributes={"a": 1, "b": 2})
+    def my_func(x, y):
+        return x + y
+
+When adding additional metadata to the trace decorator constructor, these additional components will be logged along with the span entry within 
+the trace that is stored within the active MLflow experiment.
+
+**What is captured?**
```

Comment:
Is it possible to make this a little bigger?

<img width="659" alt="image" src="https://github.com/mlflow/mlflow/assets/17039389/8526b637-169f-4de8-8636-0f5a38348afd">

Reply:
Changed to h4 and reworded the sections to feel like headers